src/App.scss

```diff
@@ -0,0 +1,21 @@
+@import "../src/common/colors.scss";
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Open Sans", sans-serif;
+}
+
+a {
+  text-decoration: none;
+  color: $font-primary;
+}
+
+body {
+  background: $primary-color;
+}
+
+.container {
+  margin: 0 40px;
+}
```

Comment:
As stated in the project guidelines kindly ensure that you follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) in terms of: 
1. Typography: font face, size, and weight
2. Layout: composition and space between elements. 

You should also remember to select one main colour for the app and be consistent.

### Your home page 

![Screenshot from 2023-01-21 16-11-56](https://user-images.githubusercontent.com/65233860/213876215-33a130df-9a41-483f-a590-8cf38605b862.png)


### The design guideline for hompage


![Screenshot from 2023-01-21 16-11-13](https://user-images.githubusercontent.com/65233860/213876226-0ddd5c44-34be-4aab-ad97-2a418f591b03.png)  

### Your details page

![Screenshot from 2023-01-21 16-11-44](https://user-images.githubusercontent.com/65233860/213876252-dcfc311a-de97-405b-9640-b0bfc5c26710.png)


### The design guidelines for details page

![Screenshot from 2023-01-21 16-11-20](https://user-images.githubusercontent.com/65233860/213876257-180e9fb2-2ff8-4203-82a5-fce7980b2f19.png)